# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../heimdal/heimdal-1.3.3-no-openssl-manpages.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Remove all manpage files which are normally installed by openssl

diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_cbc_cksum.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_cbc_cksum.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_cbc_cksum.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_cbc_cksum.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_cbc_encrypt.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_cbc_encrypt.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_cbc_encrypt.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_cbc_encrypt.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_cfb64_encrypt.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_cfb64_encrypt.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_cfb64_encrypt.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_cfb64_encrypt.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_check_key_parity.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_check_key_parity.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_check_key_parity.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_check_key_parity.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_ecb3_encrypt.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_ecb3_encrypt.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_ecb3_encrypt.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_ecb3_encrypt.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_ecb_encrypt.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_ecb_encrypt.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_ecb_encrypt.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_ecb_encrypt.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_ede3_cbc_encrypt.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_ede3_cbc_encrypt.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_ede3_cbc_encrypt.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_ede3_cbc_encrypt.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_encrypt.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_encrypt.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_encrypt.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_encrypt.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_init_random_number_generator.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_init_random_number_generator.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_init_random_number_generator.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_init_random_number_generator.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_is_weak_key.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_is_weak_key.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_is_weak_key.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_is_weak_key.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_key_sched.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_key_sched.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_key_sched.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_key_sched.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_new_random_key.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_new_random_key.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_new_random_key.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_new_random_key.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_pcbc_encrypt.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_pcbc_encrypt.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_pcbc_encrypt.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_pcbc_encrypt.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_random_key.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_random_key.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_random_key.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_random_key.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_set_key.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_set_key.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_set_key.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_set_key.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_set_key_checked.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_set_key_checked.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_set_key_checked.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_set_key_checked.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_set_key_unchecked.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_set_key_unchecked.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_set_key_unchecked.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_set_key_unchecked.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_set_odd_parity.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_set_odd_parity.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_set_odd_parity.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_set_odd_parity.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_string_to_key.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_string_to_key.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DES_string_to_key.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DES_string_to_key.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_des.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_check_pubkey.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_check_pubkey.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_check_pubkey.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_check_pubkey.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_compute_key.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_compute_key.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_compute_key.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_compute_key.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_free.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_free.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_free.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_free.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_generate_key.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_generate_key.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_generate_key.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_generate_key.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_generate_parameters_ex.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_generate_parameters_ex.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_generate_parameters_ex.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_generate_parameters_ex.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_get_default_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_get_default_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_get_default_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_get_default_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_get_ex_data.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_get_ex_data.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_get_ex_data.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_get_ex_data.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_imath_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_imath_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_imath_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_imath_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_new.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_new.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_new.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_new.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_new_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_new_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_new_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_new_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_null_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_null_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_null_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_null_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_set_default_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_set_default_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_set_default_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_set_default_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_set_ex_data.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_set_ex_data.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_set_ex_data.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_set_ex_data.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_set_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_set_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_set_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_set_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_size.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_size.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_size.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_size.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_up_ref.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_up_ref.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/DH_up_ref.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/DH_up_ref.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_dh.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_BytesToKey.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_BytesToKey.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_BytesToKey.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_BytesToKey.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_block_size.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_block_size.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_block_size.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_block_size.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_cipher.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_cipher.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_cipher.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_cipher.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_cleanup.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_cleanup.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_cleanup.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_cleanup.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_ctrl.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_ctrl.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_ctrl.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_ctrl.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_core.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_flags.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_flags.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_flags.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_flags.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_get_app_data.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_get_app_data.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_get_app_data.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_get_app_data.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_init.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_init.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_init.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_init.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_iv_length.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_iv_length.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_iv_length.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_iv_length.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_key_length.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_key_length.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_key_length.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_key_length.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_mode.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_mode.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_mode.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_mode.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_rand_key.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_rand_key.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_rand_key.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_rand_key.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_core.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_set_app_data.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_set_app_data.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_set_app_data.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_set_app_data.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_set_key_length.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_set_key_length.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_set_key_length.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_CTX_set_key_length.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_block_size.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_block_size.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_block_size.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_block_size.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_iv_length.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_iv_length.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_iv_length.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_iv_length.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_key_length.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_key_length.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_key_length.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CIPHER_key_length.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CipherFinal_ex.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CipherFinal_ex.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CipherFinal_ex.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CipherFinal_ex.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CipherInit_ex.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CipherInit_ex.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CipherInit_ex.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CipherInit_ex.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CipherUpdate.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CipherUpdate.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_CipherUpdate.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_CipherUpdate.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_Digest.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_Digest.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_Digest.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_Digest.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_DigestFinal_ex.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_DigestFinal_ex.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_DigestFinal_ex.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_DigestFinal_ex.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_DigestInit_ex.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_DigestInit_ex.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_DigestInit_ex.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_DigestInit_ex.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_DigestUpdate.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_DigestUpdate.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_DigestUpdate.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_DigestUpdate.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_block_size.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_block_size.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_block_size.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_block_size.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_cleanup.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_cleanup.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_cleanup.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_cleanup.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_create.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_create.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_create.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_create.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_destroy.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_destroy.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_destroy.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_destroy.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_init.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_init.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_init.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_init.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_md.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_md.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_md.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_md.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_size.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_size.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_size.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_CTX_size.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_block_size.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_block_size.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_block_size.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_block_size.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_size.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_size.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_MD_size.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_MD_size.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_aes_128_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_aes_128_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_aes_128_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_aes_128_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_aes_192_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_aes_192_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_aes_192_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_aes_192_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_aes_256_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_aes_256_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_aes_256_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_aes_256_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_camellia_128_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_camellia_128_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_camellia_128_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_camellia_128_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_camellia_192_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_camellia_192_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_camellia_192_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_camellia_192_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_camellia_256_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_camellia_256_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_camellia_256_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_camellia_256_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_des_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_des_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_des_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_des_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_des_ede3_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_des_ede3_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_des_ede3_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_des_ede3_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_enc_null.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_enc_null.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_enc_null.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_enc_null.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_get_cipherbyname.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_get_cipherbyname.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_get_cipherbyname.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_get_cipherbyname.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_128_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_128_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_128_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_128_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_192_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_192_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_192_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_192_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_256_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_256_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_256_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_aes_256_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_128_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_128_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_128_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_128_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_192_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_192_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_192_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_192_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_256_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_256_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_256_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_camellia_256_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_des_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_des_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_des_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_des_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_des_ede3_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_des_ede3_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_des_ede3_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_des_ede3_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md2.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md2.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md2.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md2.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md4.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md4.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md4.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md4.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md5.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md5.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md5.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_md5.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_40_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_40_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_40_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_40_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_64_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_64_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_64_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_64_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_rc2_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_sha1.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_sha1.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_sha1.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_sha1.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_sha256.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_sha256.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_sha256.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_hcrypto_sha256.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_md2.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_md2.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_md2.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_md2.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_md4.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_md4.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_md4.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_md4.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_md5.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_md5.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_md5.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_md5.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_md_null.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_md_null.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_md_null.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_md_null.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc2_40_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc2_40_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc2_40_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc2_40_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc2_64_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc2_64_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc2_64_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc2_64_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc2_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc2_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc2_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc2_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc4.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc4.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc4.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc4.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc4_40.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc4_40.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_rc4_40.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_rc4_40.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_sha.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_sha.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_sha.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_sha.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_sha1.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_sha1.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_sha1.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_sha1.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_sha256.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_sha256.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_sha256.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_sha256.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_wincrypt_des_ede3_cbc.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_wincrypt_des_ede3_cbc.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/EVP_wincrypt_des_ede3_cbc.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/EVP_wincrypt_des_ede3_cbc.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_evp.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_core.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms_conf.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms_conf.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms_conf.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms_conf.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_core.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms_noconf.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms_noconf.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms_noconf.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/OpenSSL_add_all_algorithms_noconf.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_core.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/PKCS5_PBKDF2_HMAC_SHA1.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/PKCS5_PBKDF2_HMAC_SHA1.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/PKCS5_PBKDF2_HMAC_SHA1.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/PKCS5_PBKDF2_HMAC_SHA1.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_misc.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_add.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_add.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_add.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_add.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_bytes.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_bytes.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_bytes.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_bytes.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_cleanup.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_cleanup.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_cleanup.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_cleanup.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_file_name.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_file_name.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_file_name.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_file_name.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_get_rand_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_get_rand_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_get_rand_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_get_rand_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_load_file.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_load_file.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_load_file.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_load_file.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_pseudo_bytes.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_pseudo_bytes.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_pseudo_bytes.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_pseudo_bytes.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_seed.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_seed.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_seed.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_seed.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_set_rand_engine.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_set_rand_engine.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_set_rand_engine.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_set_rand_engine.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_set_rand_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_set_rand_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_set_rand_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_set_rand_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_status.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_status.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_status.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_status.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_write_file.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_write_file.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RAND_write_file.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RAND_write_file.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rand.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_free.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_free.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_free.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_free.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rsa.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_get_app_data.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_get_app_data.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_get_app_data.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_get_app_data.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rsa.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_get_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_get_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_get_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_get_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rsa.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_new.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_new.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_new.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_new.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rsa.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_new_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_new_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_new_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_new_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rsa.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_set_app_data.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_set_app_data.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_set_app_data.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_set_app_data.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rsa.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_set_method.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_set_method.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_set_method.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_set_method.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rsa.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_up_ref.3 heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_up_ref.3
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/man/man3/RSA_up_ref.3	2010-05-27 00:03:22.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/man/man3/RSA_up_ref.3	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-.so man3/hcrypto_rsa.3
diff -ruN heimdal-1.3.3-orig/doc/doxyout/hcrypto/manpages heimdal-1.3.3/doc/doxyout/hcrypto/manpages
--- heimdal-1.3.3-orig/doc/doxyout/hcrypto/manpages	2010-05-27 00:03:23.000000000 +0200
+++ heimdal-1.3.3/doc/doxyout/hcrypto/manpages	2010-09-29 14:28:05.104206182 +0200
@@ -1,110 +1,3 @@
-hcrypto/man/man3/DES_cbc_cksum.3
-hcrypto/man/man3/DES_cbc_encrypt.3
-hcrypto/man/man3/DES_cfb64_encrypt.3
-hcrypto/man/man3/DES_check_key_parity.3
-hcrypto/man/man3/DES_ecb3_encrypt.3
-hcrypto/man/man3/DES_ecb_encrypt.3
-hcrypto/man/man3/DES_ede3_cbc_encrypt.3
-hcrypto/man/man3/DES_encrypt.3
-hcrypto/man/man3/DES_init_random_number_generator.3
-hcrypto/man/man3/DES_is_weak_key.3
-hcrypto/man/man3/DES_key_sched.3
-hcrypto/man/man3/DES_new_random_key.3
-hcrypto/man/man3/DES_pcbc_encrypt.3
-hcrypto/man/man3/DES_random_key.3
-hcrypto/man/man3/DES_set_key.3
-hcrypto/man/man3/DES_set_key_checked.3
-hcrypto/man/man3/DES_set_key_unchecked.3
-hcrypto/man/man3/DES_set_odd_parity.3
-hcrypto/man/man3/DES_string_to_key.3
-hcrypto/man/man3/DH_check_pubkey.3
-hcrypto/man/man3/DH_compute_key.3
-hcrypto/man/man3/DH_free.3
-hcrypto/man/man3/DH_generate_key.3
-hcrypto/man/man3/DH_generate_parameters_ex.3
-hcrypto/man/man3/DH_get_default_method.3
-hcrypto/man/man3/DH_get_ex_data.3
-hcrypto/man/man3/DH_imath_method.3
-hcrypto/man/man3/DH_new.3
-hcrypto/man/man3/DH_new_method.3
-hcrypto/man/man3/DH_null_method.3
-hcrypto/man/man3/DH_set_default_method.3
-hcrypto/man/man3/DH_set_ex_data.3
-hcrypto/man/man3/DH_set_method.3
-hcrypto/man/man3/DH_size.3
-hcrypto/man/man3/DH_up_ref.3
-hcrypto/man/man3/EVP_aes_128_cbc.3
-hcrypto/man/man3/EVP_aes_192_cbc.3
-hcrypto/man/man3/EVP_aes_256_cbc.3
-hcrypto/man/man3/EVP_BytesToKey.3
-hcrypto/man/man3/EVP_camellia_128_cbc.3
-hcrypto/man/man3/EVP_camellia_192_cbc.3
-hcrypto/man/man3/EVP_camellia_256_cbc.3
-hcrypto/man/man3/EVP_CIPHER_block_size.3
-hcrypto/man/man3/EVP_CIPHER_CTX_block_size.3
-hcrypto/man/man3/EVP_CIPHER_CTX_cipher.3
-hcrypto/man/man3/EVP_CIPHER_CTX_cleanup.3
-hcrypto/man/man3/EVP_CIPHER_CTX_ctrl.3
-hcrypto/man/man3/EVP_CIPHER_CTX_flags.3
-hcrypto/man/man3/EVP_CIPHER_CTX_get_app_data.3
-hcrypto/man/man3/EVP_CIPHER_CTX_init.3
-hcrypto/man/man3/EVP_CIPHER_CTX_iv_length.3
-hcrypto/man/man3/EVP_CIPHER_CTX_key_length.3
-hcrypto/man/man3/EVP_CIPHER_CTX_mode.3
-hcrypto/man/man3/EVP_CIPHER_CTX_rand_key.3
-hcrypto/man/man3/EVP_CIPHER_CTX_set_app_data.3
-hcrypto/man/man3/EVP_CIPHER_CTX_set_key_length.3
-hcrypto/man/man3/EVP_CIPHER_iv_length.3
-hcrypto/man/man3/EVP_CIPHER_key_length.3
-hcrypto/man/man3/EVP_CipherFinal_ex.3
-hcrypto/man/man3/EVP_CipherInit_ex.3
-hcrypto/man/man3/EVP_CipherUpdate.3
-hcrypto/man/man3/EVP_des_cbc.3
-hcrypto/man/man3/EVP_des_ede3_cbc.3
-hcrypto/man/man3/EVP_Digest.3
-hcrypto/man/man3/EVP_DigestFinal_ex.3
-hcrypto/man/man3/EVP_DigestInit_ex.3
-hcrypto/man/man3/EVP_DigestUpdate.3
-hcrypto/man/man3/EVP_enc_null.3
-hcrypto/man/man3/EVP_get_cipherbyname.3
-hcrypto/man/man3/EVP_hcrypto_aes_128_cbc.3
-hcrypto/man/man3/EVP_hcrypto_aes_192_cbc.3
-hcrypto/man/man3/EVP_hcrypto_aes_256_cbc.3
-hcrypto/man/man3/EVP_hcrypto_camellia_128_cbc.3
-hcrypto/man/man3/EVP_hcrypto_camellia_192_cbc.3
-hcrypto/man/man3/EVP_hcrypto_camellia_256_cbc.3
-hcrypto/man/man3/EVP_hcrypto_des_cbc.3
-hcrypto/man/man3/EVP_hcrypto_des_ede3_cbc.3
-hcrypto/man/man3/EVP_hcrypto_md2.3
-hcrypto/man/man3/EVP_hcrypto_md4.3
-hcrypto/man/man3/EVP_hcrypto_md5.3
-hcrypto/man/man3/EVP_hcrypto_rc2_40_cbc.3
-hcrypto/man/man3/EVP_hcrypto_rc2_64_cbc.3
-hcrypto/man/man3/EVP_hcrypto_rc2_cbc.3
-hcrypto/man/man3/EVP_hcrypto_sha1.3
-hcrypto/man/man3/EVP_hcrypto_sha256.3
-hcrypto/man/man3/EVP_md2.3
-hcrypto/man/man3/EVP_md4.3
-hcrypto/man/man3/EVP_md5.3
-hcrypto/man/man3/EVP_MD_block_size.3
-hcrypto/man/man3/EVP_MD_CTX_block_size.3
-hcrypto/man/man3/EVP_MD_CTX_cleanup.3
-hcrypto/man/man3/EVP_MD_CTX_create.3
-hcrypto/man/man3/EVP_MD_CTX_destroy.3
-hcrypto/man/man3/EVP_MD_CTX_init.3
-hcrypto/man/man3/EVP_MD_CTX_md.3
-hcrypto/man/man3/EVP_MD_CTX_size.3
-hcrypto/man/man3/EVP_md_null.3
-hcrypto/man/man3/EVP_MD_size.3
-hcrypto/man/man3/EVP_rc2_40_cbc.3
-hcrypto/man/man3/EVP_rc2_64_cbc.3
-hcrypto/man/man3/EVP_rc2_cbc.3
-hcrypto/man/man3/EVP_rc4.3
-hcrypto/man/man3/EVP_rc4_40.3
-hcrypto/man/man3/EVP_sha.3
-hcrypto/man/man3/EVP_sha1.3
-hcrypto/man/man3/EVP_sha256.3
-hcrypto/man/man3/EVP_wincrypt_des_ede3_cbc.3
 hcrypto/man/man3/hcrypto_core.3
 hcrypto/man/man3/hcrypto_des.3
 hcrypto/man/man3/hcrypto_dh.3
@@ -112,32 +5,8 @@
 hcrypto/man/man3/hcrypto_misc.3
 hcrypto/man/man3/hcrypto_rand.3
 hcrypto/man/man3/hcrypto_rsa.3
-hcrypto/man/man3/OpenSSL_add_all_algorithms.3
-hcrypto/man/man3/OpenSSL_add_all_algorithms_conf.3
-hcrypto/man/man3/OpenSSL_add_all_algorithms_noconf.3
 hcrypto/man/man3/page_des.3
 hcrypto/man/man3/page_dh.3
 hcrypto/man/man3/page_evp.3
 hcrypto/man/man3/page_rand.3
 hcrypto/man/man3/page_rsa.3
-hcrypto/man/man3/PKCS5_PBKDF2_HMAC_SHA1.3
-hcrypto/man/man3/RAND_add.3
-hcrypto/man/man3/RAND_bytes.3
-hcrypto/man/man3/RAND_cleanup.3
-hcrypto/man/man3/RAND_file_name.3
-hcrypto/man/man3/RAND_get_rand_method.3
-hcrypto/man/man3/RAND_load_file.3
-hcrypto/man/man3/RAND_pseudo_bytes.3
-hcrypto/man/man3/RAND_seed.3
-hcrypto/man/man3/RAND_set_rand_engine.3
-hcrypto/man/man3/RAND_set_rand_method.3
-hcrypto/man/man3/RAND_status.3
-hcrypto/man/man3/RAND_write_file.3
-hcrypto/man/man3/RSA_free.3
-hcrypto/man/man3/RSA_get_app_data.3
-hcrypto/man/man3/RSA_get_method.3
-hcrypto/man/man3/RSA_new.3
-hcrypto/man/man3/RSA_new_method.3
-hcrypto/man/man3/RSA_set_app_data.3
-hcrypto/man/man3/RSA_set_method.3
-hcrypto/man/man3/RSA_up_ref.3
